client/src/components/profile.ts

```diff
@@ -602,7 +599,8 @@ const input: ProfilePromptInput = {
   [ProfilePromptType.ClientId]: {
     title: "Client ID",
     placeholder: "Enter a client ID",
-    description: "Enter the registered client ID. An example is myapp.client.",
+    description:
```

Comment:
Hey @tsej. I believe client id info has been covered in https://github.com/sassoftware/vscode-sas-extension#profile-anatomy-viya. 

That said, it isn't immediately apparent what you should enter for client id without looking at the readme.